source/vistas/core/color.py

```diff
@@ -117,7 +123,7 @@ def hsva_list(self):
 
 
 def interpolate_color(range, min_color: Color, max_color: Color, t):
-    """ Linearly interpolate in the HSV color space """
+    """ Linearly interpolate in the HSV color space. Copies colors in RGB space to HSV. """
```

Comment:
Although, I guess that's already implied by "in the HSV color space." In which case maybe the second sentence isn't required at all.